### ISSUE TITLE
openssl CVE in grafana docker image

### ISSUE THREAD

**Issue by tooptoop4** [grafana/grafana#57939]
https://www.trendmicro.com/en_us/research/22/j/openssl-critical-security-vulnerability-fix.html

**Comment by gamethis** [comment]
I have validated by just rebuilding at the latests tag that it is fixed if you just rebuild.
If the release had waited till the release was done this morning it would have had it in it

**Comment by EAlf91** [comment]
@gamethis how did you check on this? Latest tag still uses openssl 3.0.5 which should be affected by this, right?

**Comment by EAlf91** [comment]
Hey @gamethis how did you check on this? To my understanding latest still uses openssl 3.0.5, which should be affected by this

**Comment by larntz** [comment]
We are also tracking this hoping for a new image with a remediated openssl.

**Comment by gamethis** [comment]
> Hey @gamethis how did you check on this? To my understanding latest still uses openssl 3.0.5, which should be affected by this

@EAlf91 I verified that the latest public image had the openssl 3.0.5 in it as you said.
then I pulled the repo, switched tot he latest release tag 9.2.3. Performed a docker build, validated that the openssl had updated to 3.0.7.

I have used the image in my infrastructure and validated all works well with the patched version of openssl in it.

I'm looking forward to the grafana team releasing a fixed official version hopefully soon.
